controller/src/main/java/org/jboss/as/controller/PrimitiveListAttributeDefinition.java

```diff
@@ -168,7 +172,7 @@ public Builder(final String name, final ModelType valueType) {
 
         public Builder(final PrimitiveListAttributeDefinition basis) {
             super(basis);
-            this.valueType = basis.getValueType();
+            this.valueType = basis.getValueAttributeDefinition().getType();
```

Comment:
is that change requrired? The `getValueType()` method is still there.

Reply:
Yes :) I don't remember why but otherwise it breaks